src/parser/rogue/outlaw/modules/spells/RollTheBonesCounter.js

```diff
@@ -16,7 +16,7 @@ class RollTheBonesCounter extends Analyzer {
 
   constructor(...args) {
     super(...args);
-    this.active = !this.selectedCombatant.hasTalent(SPELLS.SLICE_AND_DICE_TALENT.id);
+    this.active = true
```

Comment:
If the module should always be loaded, you dont need the this.active to be set to true.